resources/lang/hu/rooms.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    'rooms' => 'Szobabeosztás',
+    'search' => 'Keresés',
+    'search_name' => 'Keress névre vagy szobaszámra',
+    'resident1' => '1. lakó',
```

Comment:
you can do this with an indexed loop, add only 'resident' here

Reply:
English won't work that way.